packages/contracts/contracts/L1/AssetManager.sol

```diff
@@ -241,6 +241,18 @@ contract AssetManager is ISemver, IERC721Receiver, IAssetManager {
         return _vaults[validator].asset.validatorRewardKro;
     }
 
+    /**
+     * @inheritdoc IAssetManager
+     */
+    function canUndelegateKroAt(
```

Comment:
How about moving this function right below `previewUndelegate`? I think it's better to locate the functions related to delegators nearly.